README.Rmd

```diff
@@ -85,13 +85,6 @@ watch the clusters fuse as the regularization level is increased:
 plot(carp_fit, type = "path", dynamic = TRUE)
 ```
 
-`clustRviz` also provides interactive dendrograms using the
-[`heatmaply`](https://cran.r-project.org/package=heatmaply) package: 
-
-```{r carp_heatmap}
-plot(carp_fit, type = "heatmap", interactive = TRUE)
-```
-
 ### BiClustering
 
 The use of `CBASS` for convex biclustering is similar, and we demonstrate it here
```

Comment:
Here `plot(cbass_fit, k.row = 3)` and `plot(cbass_fit, type = "heatmap", k.row = 3)` produce the same plot. Do we need to omit one of them?

Reply:
Good catch - will fix!